<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dharanidhar-PC" failures="1" tests="3" name="StandAloneTest" time="12.678" errors="0" timestamp="2025-05-04T12:03:16 IST">
  <testcase classname="dharani.test.StandAloneTest" name="SubmitOrder" time="4.81"/>
  <testcase classname="dharani.test.StandAloneTest" name="SubmitOrder" time="3.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at dharani.test.StandAloneTest.SubmitOrder(StandAloneTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- SubmitOrder -->
  <testcase classname="dharani.test.StandAloneTest" name="OrderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- StandAloneTest -->
